frontend/src/features/env/AdminFeedbackModal.tsx

```diff
@@ -68,7 +68,9 @@ export const AdminFeedbackModal = ({
   const feedbackMutation = useFeedbackMutation()
 
   const handleSubmitForm = handleSubmit((formInputs: AdminFeedbackFormDto) => {
-    if (!feedbackForm) return
+    if (!feedbackForm) {
```

Comment:
nit: but is this change required?

Reply:
[20bd598](https://github.com/opengovsg/FormSG/pull/5981/commits/20bd598415531c3246827887baed86276c104147)